erpnext/stock/doctype/stock_entry/stock_entry.json

```diff
@@ -638,12 +639,19 @@
    "label": "Harvest",
    "options": "Harvest",
    "read_only": 1
+  },
+  {
+   "fetch_from": "work_order.purchase_receipt",
+   "fieldname": "purchase_receipt",
+   "fieldtype": "Data",
```

Comment:
field type should be read only

Reply:
filed type is read-only see line number 647 and 648